src/reflect/scala/reflect/internal/StdNames.scala

```diff
@@ -598,6 +598,7 @@ trait StdNames {
     val Annotation: NameType           = "Annotation"
     val Any: NameType                  = "Any"
     val AnyVal: NameType               = "AnyVal"
+    val AppliedTypeTree: NameType      = "AppliedTypeTree"
```

Comment:
Ooops, this is detritus from my first attempt, I'll force push a removal of this line.